biomodels/README.md

```diff
@@ -0,0 +1,62 @@
+# BioModels Data Insertion Release Step
+
+The BioModels step is run following the <a href="https://github.com/reactome/analysis-core">analysis-core</a> and before UpdateGKCurrent (link needed) in Reactome's quarterly data release. It consists of two parts: (1) running the <a href="https://github.com/reactome/biomodels-mapper">biomodels-mapper</a> to generate `models2pathways.tsv` and than (2) running the code found in this repository to perform database updates. This is a rewrite of <a href="https://github.com/reactome/Release/blob/master/scripts/release/biomodels/add_links_to_biomodels_resource.pl">add_links_to_biomodels_resource.pl</a>
+
+This tool will insert the cross-references for the BioModels Reference Database into the appropriate pathway instances of the release database.
+
+It will:
+
+ - Read the models2pathways.tsv file describing the relationships between Reactome pathways and BioModels identifiers.
+ - Create a BioModels Reference Database instance (if one does not already exist)
+ - Create the Database Identifier instances representing the BioModels identifiers and insert them into Pathway instances under the 'crossReference' attribute
+ 
+ ```
+## models2pathways.tsv example
+BIOMD0000000498 R-HSA-917977    8.191346134722721E-7    https://reactome.org/PathwayBrowser/#/R-HSA-917977      Transferrin endocytosis and recycling   IEA     Homo sapiens
+BIOMD0000000287 R-HSA-69563     6.066213418698396E-6    https://reactome.org/PathwayBrowser/#/R-HSA-69563       p53-Dependent G1 DNA Damage Response    IEA     Homo sapiens
+BIOMD0000000287 R-HSA-2559586   6.066213418698396E-6    https://reactome.org/PathwayBrowser/#/R-HSA-2559586     DNA Damage/Telomere Stress Induced Senescence   IEA     Homo sapiens
+```
+
+For the purposes of this module, the first two columns are the most important. The first column contains BioModels identifiers, and the second column contains Reactome Pathway identifiers that have have been connected to them by biomodels-mapper. For each line in `models2pathways.tsv`, the corresponding Pathway instance in Reactome's relationship database will receive a CrossReference that points to the biomodel identifier. If the biomodel identifier does not already have a database instance, it will be created before being slotted into the Pathway instance as an instance of type `DatabaseIdentifier`. All biomodels identifier instances are connected to the sole 'BioModels Database' `ReferenceDatabase` object. 
```

Comment:
For the biomodels-mapper mentioned at the end of the second sentence, it would be great to have a link to its GitHub repository: https://github.com/reactome/biomodels-mapper

Reply:
I linked to it earlier in the README but couldn't hurt to link again

Reply:
Ah right - I didn't notice since the review shows the HTML instead of the formatted link.  It's probably okay not to link it again.  Maybe just add in brackets "see link above"?